package.json

```diff
@@ -13,6 +13,7 @@
     "build": "webpack --config webpack.production.js",
     "i18n:manage": "node src/translations/runner.js",
     "test": "jest --coverage --color",
+    "test:interactive": "jest --watch --coverage --color",
```

Comment:
This tool is ... interesting. I'm not sure if it made things faster for me or not, but it's worth trying out!